validator_client/src/signing_method.rs

```diff
@@ -182,6 +184,10 @@ impl SigningMethod {
                         Web3SignerObject::RandaoReveal { epoch }
                     }
                     SignableMessage::BeaconBlock(block) => Web3SignerObject::beacon_block(block)?,
+                    SignableMessage::BlobSidecar(_) => {
+                        // https://github.com/ConsenSys/web3signer/issues/726
+                        unimplemented!("Web3Signer blob signing not implemented.")
```

Comment:
nice, they just released it!

Reply:
Spent time trying to get the tests working, but the web3signer seems to need `DENEB_FORK_EPOCH` to sign anything from deneb. From what I can tell this can't be manually configured, we have to use one of its built in networks, none of which have a valid `DENEB_FORK_EPOCH` epoch set. So I think this will have to wait a bit longer.

Reply:
If I remember correctly, I think it only uses the built-in networks fork 
 version values and doesn't actually check the epoch matches its built in networks fork epochs - I was able to get web3signer working on a local testnet that has matching fork version values as one of the built in networks (so it produces valid signatures). It might still be possible to test this if we modify `DENEB_FORK_EPOCH` value of the default spec, unless they've added validation to fork epochs since I last used it, although this could probably wait.

Reply:
Just noting that the issue for this is here: #4453.